pulser-core/pulser/backend/remote.py

```diff
@@ -161,3 +171,29 @@ def _type_check_job_params(job_params: list[JobParams] | None) -> None:
                     "All elements of 'job_params' must be dictionaries; "
                     f"got {type(d)} instead."
                 )
+
+    def open_batch(self) -> _OpenBatchContextManager:
+        """Create an open batch within a context manager object"""
```

Comment:
nit: Not sure if the linter caught it, but the style guide we follow needs a couple modifications:
```suggestion
        """Creates an open batch within a context manager object."""
```